Mathlib/Data/Matrix/Basic.lean

```diff
@@ -1684,17 +1686,27 @@ def mulVec [Fintype n] (M : Matrix m n α) (v : n → α) : m → α
   | i => (fun j => M i j) ⬝ᵥ v
 #align matrix.mul_vec Matrix.mulVec
 
+/- The precedence of 73 comes immediately before ` ⬝ᵥ ` for `Matrix.dotProduct`
+   so that `A *ᵥ v ⬝ᵥ B *ᵥ w` is parsed as `(A *ᵥ v) ⬝ᵥ (B *ᵥ w)` here. -/
```

Comment:
I think we should update the `mulVec` docstring and move this precedence note into it. Maybe
```
/-- 
`M *ᵥ v` (notation for `mulVec M v`) is the matrix-vector product of a matrix `M` and vector `v`,
where `v` is seen as a column vector.
Put another way, `M *ᵥ v` is the vector whose entries
are those of `M * col v` (see `col_mulVec`).

The notation has precedence 73, which comes immediately before ` ⬝ᵥ ` for `Matrix.dotProduct`,
so that `A *ᵥ v ⬝ᵥ B *ᵥ w` is parsed as `(A *ᵥ v) ⬝ᵥ (B *ᵥ w)`.
-/
```
Similarly for `vecMul`.

Reply:
Done. Please doublecheck.

Reply:
Thanks, looks good